RELEASE/scripts/autoscend/auto_util.ash

```diff
@@ -1433,6 +1447,15 @@ boolean cloverUsageInit()
 	if(auto_haveAugustScepter() && get_property("_augSkillsCast").to_int() < 5 && !get_property("_aug2Cast").to_boolean())
 	{
 		use_skill($skill[Aug. 2nd: Find an Eleven-Leaf Clover Day]);
+		if(have_effect($effect[Lucky!]) > 0)
```

Comment:
Good catch. Also feel like scepter should be used before clovers. Scepter lucky charge is lost if we don't use it in a given day. Can stockpile clovers

Reply:
Good point. I'll move it up. The account I'm running doesn't have Sceptre or BoFa (yet) so I can't test using this myself though.